docs/index.md

```diff
@@ -40,15 +40,17 @@ mv googletest-release-1.7.0 gtest-1.7.0
 cd ..
 mkdir build
 cd build
-cmake -DARGO_BACKEND_MPI=ON        \
-      -DARGO_BACKEND_SINGLENODE=ON \
-      -DARGO_TESTS=ON              \
-      -DBUILD_DOCUMENTATION=ON     \
-      -DCMAKE_CXX_COMPILER=mpic++  \
-      -DCMAKE_C_COMPILER=mpicc     \
+cmake -DARGO_BACKEND_MPI=ON              \
+      -DARGO_BACKEND_SINGLENODE=ON       \
+      -DARGO_TESTS=ON                    \
+      -DBUILD_DOCUMENTATION=ON           \
+      -DCMAKE_CXX_COMPILER=mpic++        \
+      -DCMAKE_C_COMPILER=mpicc           \
+      -DCMAKE_INSTALL_PREFIX=/usr/local/ \
       ../
 make
 make test
+make install
```

Comment:
I am not sure I like the idea of having the `make install` in the basic instructions since, presumably, more users will not have root permissions on their cluster. I might be wrong about the "most users" part though. What is the common practice for open source applications?

Reply:
Given current usage, I think "most users" will replace the path with one in their home directory. @lundgren87 can probably explain better why they started using `make install` to begin with, but it appears that using stuff from the build tree is less standard practice than changing install paths and `make install`.

From my limited experience the default prefix seems to be `/usr/local/` in most projects, even when root privileges are not required for the software. The documentation patch already adds the option that needs to be changed to change the path (4 lines above the quoted line), do you think it should be phrased in a way that nudges people into actually changing it?

Reply:
Maybe adding something along the lines of "Note that you might need root access to install to the /usr/local/ directory. If you do not have permission to do this on your system, you should change the parameter to a directory where you have access to. Please contact your system administrator for more information."?

Maybe without the "contact your admin line", since it's already mentioned somewhere else.

Reply:
Including a `make install` is sort of standard practice in most code bases and the default prefix is indeed `/usr/local`.  Most code bases are mute about permissions or `sudo`.  The only difference is that most code bases I am aware of also include a `configure` step where one can specify an appropriate `--prefix` (if `/usr/local` is not the desired one).  See for example [CVC4's instructions](https://github.com/CVC4/CVC4/blob/master/INSTALL.md).

Rather than including the verbose text of @SakalisC, I suggest we just make sure that the instructions explicitly mention how one specifies a different prefix.

Reply:
see new version 4a26f57